.github/workflows/docker.yml

```diff
@@ -7,15 +7,11 @@ on:
     - develop
 
 jobs:
-  build_docker_images:
+  build_docker_image:
     # This workflow is only of value to PyBaMM and would always be skipped in forks
     if: github.repository_owner == 'pybamm-team'
-    name: Image (${{ matrix.build-args }})
+    name: Build image
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        build-args: ["No solvers", "JAX", "IDAKLU", "ALL"]
-      fail-fast: true
```

Comment:
I'd suggest to only remove - 
```
matrix:
    build-args: ["No solvers", "JAX", "IDAKLU", "ALL"]
```

Reply:
We'll have to remove the `strategy:` as well because that is applicable just for job matrices